### PR TITLE
fixes : twitter icon

### PR DESCRIPTION
**Pull request by Yashh56**
A new Twitter icon added  #1890

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -4898,7 +4898,7 @@
                 "original"
             ]
         },
-        "color": "#1da1f2",
+        "color": "#ffffff",
```
Comment (canaleal) [inline devicon.json:4901]: The twitter X logo's most prominent color is `#000000`  instead of white.

File: icons/twitter/twitter original.svg
Comment (canaleal) [inline icons/twitter/twitter original.svg:1]: In here, the name of the updated icon should be `twitter-original.svg`. Like shown [here](https://github.com/devicons/devicon/wiki/Organizing-SVGs).

The updated SVG must have a viewbox of `viewBox="0 0 128 128"` with the icon resized to fit the container as much as possible as shown in the [guidelines](https://github.com/devicons/devicon/wiki/SVG-Standards). 

When loading the new twitter SVG into inkscape, the svg is `viewBox="0 0 1668.56 1221.19"` and the X is not resized to reach the edges.

**Comment by canaleal** [comment]
Also, regardless of if you're fixing a bug, adding a new feature/icon, all changes are made to the develop branch.
https://github.com/devicons/devicon/wiki/Overview-on-Submitting-Icons

**Comment by Yashh56** [comment]
I will make this changes ASAP ;)

**Comment by Yashh56** [comment]
Hey @canaleal Sir, I made the required please check and review the PR

**Comment by Yashh56** [comment]
Hey Maintainer Check the PR !!

**Comment by canaleal** [comment]
Hey @Yashh56 , the SVG does not seem to be showing in the `twitter-original.svg` file. I can see the paths inside the file, but I think the transform is not allowing them to show properly. 

If possible, could you try the following:

`
<svg width="128" height="128" viewBox="0 0 128 128" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M75.916 54.2 122.542 0h-11.05L71.008 47.06 38.672 0H1.376l48.898 71.164L1.376 128h11.05L55.18 78.303 89.328 128h37.296L75.913 54.2ZM60.782 71.79l-4.955-7.086-39.42-56.386h16.972L65.19 53.824l4.954 7.086 41.353 59.15h-16.97L60.782 71.793Z" fill="#000" style="stroke-width:.104373"/></svg>
`

Also, don't forget that the branch you need to merge into is not `devicons:master`, but `devicons:develop`.

**Comment by Yashh56** [comment]
I made these changes @canaleal

**Comment by Yashh56** [comment]
Hey @canaleal Sir Check the New PR

**Pull request closed without merging**